decidim-bulletin_board-js/src/vote/vote.component.js

```diff
@@ -35,9 +35,13 @@ export class VoteComponent {
    * @method bindEvents
    * @param {Object} eventCallbacks - An object that contains event callback functions.
    * - {Function} onSetup - a function that is called when the voter is set up.
-   * - {Function} onBindStartButton - a function that receives a callback function that will be called when
-   *                                  the vote must be started.
+   * - {Function} onBindEncryptButton - a function that receives a callback function that will be called when encrypting the vote must be started
+   * - {Function} onVoteEncryption - a function that is called when the vote gets encrypted
+   * - {Function} castOrAuditBallot - a function that is called to cast or audit a ballot
    * - {Function} onStart - a function that is called when the vote has started.
+   * - {Function} onAuditVote - a function that is called to audit a vote before encrypting it.
+   * - {Function} onVoteAudition - a functio, that is called when the auditable vote is ready.
```

Comment:
```suggestion
   * - {Function} onVoteAudition - a function, that is called when the auditable vote is ready.
```

Reply:
Gosh, thanks for finding that before @orlera does. 😂